index.js

```diff
@@ -1,6 +1,6 @@
 function stringIncludes(haystack, needle) {
-  for (let i = 0; i < haystack.length; i++) {
-    if (haystack.slice(i, needle.length) === needle) {
+  for (let i = 0; i < haystack.length + 1; i++) {
+    if (haystack.slice(i, needle.length + 1) === needle) {
```

Comment:
This should be 

```JavaScript
if (haystack.slice(i, needle.length + i) === needle) {
```

because you need to move the end you are grabbing by `i` as well so you are always grabbing the right amount of letters.